mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -728,7 +728,7 @@ def dict_repr(self):
 class DetectorDistanceHOMediator(HOMediatorBase):
     def __init__(self, ho, name=''):
         super(DetectorDistanceHOMediator, self).__init__(ho, name)
-        ho.connect("positionChanged", self.value_change)
+        #ho.connect("positionChanged", self.value_change)
```

Comment:
Well at the moment we don't, its related to what I wrote at the end of my message. We need to find a good way to aggregate the resolution and the detector distance. I added the comment since it was causing me some problems when I was testing, its possible that you are not experiencing the same problem. But we will of course have a way to get those updates